### ISSUE TITLE
[FEATURE REQUEST] Add Support for AtCoder & CodeChef

### ISSUE THREAD

**Issue by Siddhant-K-code** [IEEE-NITK/CodePal#47]
- [ ] Atcoder
- [ ] Codechef

**Comment by Siddhant-K-code** [comment]
@aditya5558 are we working on this?

**Comment by adityachirania** [comment]
Hi actually we really might need some planning to add support for codechef and atcoder. As far as im aware, codechef and atcoder dont have a public api so we might only have to rely on scrapping . Will let you know when we discuss about it next !

**Comment by Siddhant-K-code** [comment]
We Can Use [This](https://github.com/online-judge-tools/api-client/blob/master/onlinejudge/service/atcoder.py) for **AtCoder**

**Comment by adityachirania** [comment]
Oh this actually seems veryy good. Thanks alot for this, we'd surely work on including this ahead now! I think codechef often has server issues and may turn out to be very slow im not sure of codechef but atcoder is definetly something we can add

**Comment by Siddhant-K-code** [comment]
> Oh this actually seems very good. Thanks alot for this, we'd surely work on including this ahead now! I think codechef often has server issues and may turn out to be very slow im not sure of codechef but atcoder is definetly something we can add

Yeah, I think Codechef can be stressful for them also. If you need any help or I can contribute to anything for AtCoder, let me know.
